# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# Filename: target/mnemosyne/pkg_patch/udhcp.patch
# Copyright (C) 2002 - 2006 Alejandro Mery
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

--- ./Makefile.orig	2006-07-19 21:51:07.000000000 +0200
+++ ./Makefile	2006-07-19 21:51:24.000000000 +0200
@@ -85,11 +85,6 @@
 else
 	$(INSTALL) -m 755 $(BOOT_PROGRAM) $(USRSBINDIR)
 endif
-	mkdir -p $(USRSHAREDIR)/udhcpc
-	for name in bound deconfig nak renew script ; do \
-		$(INSTALL) -m 755 samples/sample.$$name \
-			$(USRSHAREDIR)/udhcpc/default.$$name ; \
-	done
 	mkdir -p $(USRSHAREDIR)/man/man1
 	$(INSTALL) -m 644 dumpleases.1 $(USRSHAREDIR)/man/man1
 	mkdir -p $(USRSHAREDIR)/man/man5
